packages/ui-kit/src/components/PieChart/PieChart.tsx

```diff
@@ -0,0 +1,326 @@
+            type: variant,
+            data: values,
+            backgroundColor: chartColorPlatte,
+            ...datasets
+          })
+
+          chart.options.plugins = {
+            ...chart.options.plugins,
+            ...customPlugins
+          }
+
+          chart.update()
+          return
+        }
+
+        let config: ChartConfiguration<'pie' | 'doughnut'> = {
+          ...chartConfig,
+          type: variant,
+          data: {
+            labels: labels,
+            datasets: [
+              {
+                data: values as number[],
+                backgroundColor: chartColorPlatte,
+                borderWidth: 0,
+                offset: 4,
+                hoverOffset: 20,
+                ...datasets
+              }
+            ]
+          },
+          options: {
+            ...chartConfig.options,
+            indexAxis: 'y',
+            responsive: true,
+            maintainAspectRatio: false,
+            layout: {
+              padding: 4
+            },
+            plugins: {
+              ...chartConfig.options?.plugins,
+              ...customPlugins
+            },
+            scales: {
+              x: {
+                display: false
+              },
+              y: {
+                display: false
+              }
+            }
+          },
+          plugins: [customCanvasBackgroundColor, customChartLabelsPlugin]
+        }
+
+        if (chartConfigProps) {
+          config = chartConfigProps(config)
+        }
+
+        chartRef.current = new ChartJS(canvasRef.current, config) as ChartJS
+        canvasRef.current.style.borderRadius = '0px'
+      },
+      [variant, theme, card, chartProps, chartConfig, isDoughnut, chartConfigProps]
+    )
+
+    const destroyChart = () => {
+      if (chartRef.current) {
+        chartRef.current.destroy()
+        chartRef.current = null
+      }
+    }
+
+    // Calculate the other value and add it to the leaderboardData
+    const fetchedData = React.useMemo(() => {
+      if (!leaderboardData || !counterData) {
+        return
+      }
+
+      let leaderboardSum = 0
+
+      for (const row of leaderboardData?.leaderboard?.rows ?? []) {
+        leaderboardSum += parseFloat(row[1] as string)
+      }
+
+      const otherValue = parseFloat(counterData?.counter?.value ?? '0') - leaderboardSum
+      leaderboardData?.leaderboard?.rows.push(['Other', otherValue.toString()])
+
+      return leaderboardData
+    }, [leaderboardData, counterData])
+
+    const loadingStyles = {
+      opacity: isLoading || isLoadingStatic ? '0.3' : '1',
+      transition: 'opacity 0.2s ease-in-out'
+    }
+
+    // @TODO: we should abstract this logic to a hook
+    React.useEffect(() => {
+      function handlePropsMismatch() {
+        if (isStatic && !headers && !rows) {
+          // console.error('InvalidPropsError: You must pass either `headers` and `rows` or `query` props') we will set logs as a feature later
```

Comment:
Maybe we should start using the logger for those? @markandrus wdyt

Reply:
Yes, I think so, but let's do it in a follow-up PR.

Reply:
Planned in a follow-up PR.